core-services/secret-mirroring/_mapping.yaml

```diff
@@ -17,6 +17,12 @@ secrets:
   to:
     namespace: test-credentials
     name: cluster-secrets-openstack-vexxhost
+- from:
+    namespace: shiftstack-secrets
+    name: cloud-credentials
+  to:
+    namespace: test-credentials
+    name: cluster-secrets-openstack
```

Comment:
Added this here for ilustration, I think we probably have to do this in to steps? 
or at least create the cloud-credentials  secret under shiftstac-secrets 
Also why the different names?  why not copy shiftstack-secrets/cloud-credentials to test-credentials/cloud-credentials. 
I know names don't matter but if they dont, then make it obvious...

Reply:
Actully added the cloud-credential secret with both current cloud providers
labelred as 
moc
and
vexxhost

also added an "openstack" for default